app/assets/javascripts/topbar.js

```diff
@@ -48,19 +41,9 @@ $(document).on('mouseleave','.loc-submenu', function(){
 });
 
 function mark_active_menu() {
-  var menus = $('.menu_tab_dropdown'),
-      path = window.location.pathname + window.location.search,
-      link = $("[href='%s']".replace('%s', path));
-
-  menus.removeClass('active');
-  $("[class^='menu_tab_']").removeClass('active');
```

Comment:
Not quite sure why this was needed in the first place, couldn't find any case where the `active` class was set on the wrong element. most likely dead code, this line was here since 2013 at least.